### ISSUE TITLE
Top border of sim flashes after exiting full screen mode with Chromebook

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/function-builder#144]
**Test device**
Chromebook version 98.0.475891

**Problem description**
For https://github.com/phetsims/qa/issues/766 and https://github.com/phetsims/qa/issues/767

I noticed that after exiting full screen mode, when I send a card through the tunnel in either direction, the top border between the sim and toolbars flashes. 

I could not reproduce this on MacOS12 + safari/chrome.

**Steps to reproduce**
1. Go to any screen and select Full Screen mode in the PhET menu
2. Exit out of Full Screen mode
3. Send a card through the tunnel

Once you switch screens, the flashing stops (until you repeat the above steps again). 

**Visuals**

https://user-images.githubusercontent.com/87318828/153653810-95120223-b3b5-44a2-ae61-2061ecfd701f.mov

<details>

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Function Builder‬
URL: https://phet-dev.colorado.edu/html/function-builder/1.2.0-dev.6/phet/function-builder_all_phet.html
Version: 1.2.0-dev.6 2022-02-08 17:45:56 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.80 Safari/537.36
Language: en-US
Window: 1145x555
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by amanda-phet** [comment]
@Nancy-Salpepi did you check any other sims? This might not be a function builder bug.

**Comment by Nancy-Salpepi** [comment]
I looked at a few sims with light backgrounds where I was capable of moving things: GFL:B, Fractions:Intro, Masses and Springs, and Pendulum lab --I didn't see any flashing, although a couple of times the top border disappeared when exciting full screen mode and then reappeared once I clicked on something. 

I do see the flashing happening in the published version of function builder with the Chromebook. 
It might just be a Chromebook thing and not a function builder bug.
